inference-engine/src/mkldnn_plugin/mkldnn_graph.h

```diff
@@ -204,6 +204,7 @@ class MKLDNNGraph {
     friend InferenceEngine::CNNNetwork dump_graph_as_ie_ngraph_net(const MKLDNNGraph &graph);
 
 private:
+    void EnforceBF16();
```

Comment:
@dmitry-gorokhov Actually shouldn't we move all the code under **protected** to **private** section?

Reply:
@EgorDuplensky most likely we should. Lets do as part of bigger MKLDNNNode refactoring.